tests/gtest/Makefile.am

```diff
@@ -7,6 +7,7 @@ gtest_CPPFLAGS = \
 	-I$(top_srcdir)/src/vma \
 	-I$(top_srcdir)/tests/gtest
 
+CXXFLAGS      := $(filter-out -Wundef, $(CXXFLAGS))
```

Comment:
This doesn't look related to the current commit. If you require it later, please move it to the corresponding commit.